### ISSUE TITLE
Use case: sub numbers out for words

### ISSUE THREAD

**Issue by timhwang21** [broofa/BroofaJS#5]
Hello! Big fan of the syntax of this library over other alternatives.

I ran into a use case where I want to print strings like:

1. 0: "You have no foxes."
2. n, where 0 < n < max, where max is an arbitrarily passed number: "You have n foxes."
3. max: "You have all foxes."

While this would be quite useful, I couldn't think of a way to have a functionality like this coexist with the existing incredibly simple API. Was wondering what your thoughts were on this? Thanks!

**Comment by broofa** [comment]
I've run into "zero" case so, yeah, it'd be nice if there were a way of handling that.  The "max" case seems less common and more likely to require special handling beyond just this message.  E.g. if you have "all" of something, then the UI is probably doing other things like tearing down a progress meter, disabling a cancel button, closing a modal panel... whatever.  Also, the phrasing there is going to vary a bit more.  E.g. "Congratulations, you got all the foxes!", or "No more foxes left to find", etc.

That said, the best solution I can come up with off at the moment would be to allow for array values that supply `[quantity, label]` where `quantity` determines plurality, and `label` is the string to display for the value.  E.g.
```
simplur`You have ${[val, val == 0 ? 'no' : val >= 5 ? 'all' : val]} fox[|es]`
```

- `val == 0` -> 'You have no foxes`
- `val == 1` -> 'You have 1 fox`
- `val == 2` -> 'You have 2 foxes`
- `val == 5` -> 'You have all foxes`

But even this feels like a bit of a stretch.  I think the zero case suffers from the same issue as max.  Having zero of something... you tend to want to express that differently, leading to code like:
```
val == 0 ? 'No more foxes' : simplur(`You have ${val} fox[|es]`)
```
TL;DR: Not convinced the added complexity is worth it, but I could be convinced.

**Comment by broofa** [comment]
Went ahead and added support for this in v1.1.0.  See https://github.com/broofa/BroofaJS/blob/master/simplur/README.md#custom-quantity-strings

**Comment by timhwang21** [comment]
Awesome! For what it's worth, my use case was for a label for a checkbox group widget. ("No items selected" / "N items selected" / "All items selected")
